activemq/ci/activemq.rake

```diff
@@ -0,0 +1,88 @@
+require 'ci/common'
+
+def activemq_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def activemq_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/activemq_#{activemq_version}"
+end
+
+def activemq_dir
+  if ENV['TRAVIS']
+    return activemq_rootdir
+  end
+  if ENV['VOLATILE_DIR']
+    "#{ENV['VOLATILE_DIR']}/activemq"
+  else
+    "/tmp/integration-sdk-testing/activemq"
+  end
+end
+
+namespace :ci do
+  namespace :activemq do |flavor|
+    task before_install: ['ci:common:before_install'] do
+      sh %(docker stop dd-test-activemq || true)
+      sh %(docker rm dd-test-activemq || true)
+      sh %(mkdir -p #{activemq_dir}/data)
+      sh %(curl -s -L\
+       -o #{activemq_dir}/kahadb.tar.gz\
+       https://s3.amazonaws.com/dd-agent-tarball-mirror/apache-activemq-kahadb.tar.gz)
+      sh %(tar zxf #{activemq_dir}/kahadb.tar.gz\
+       -C #{activemq_dir}/data)
+      sh %(rm -rf #{activemq_dir}/data/kahadb/lock)
+      sh %(chmod -R a+rwxX #{activemq_dir})
```

Comment:
I don't like this, but it encounters a permissions error if I don't put this here.